docs/source/developer_documentation.rst

```diff
@@ -1,8 +1,6 @@
 Developer Documentation
 =======================
 
-**Note this document is a work in progress. Contrbitions to all sections are welcome.**
```

Comment:
Is this removal intentional?

Reply:
I think this is inherited from previous versions of the commit used as a baseline for this PR. No strong opinion here :).

Reply:
I'd suggest adding it back in.